packages/relay/src/services/VoteService.ts

```diff
@@ -35,8 +36,11 @@ export class VoteService {
      * The user can vote or cancel the vote to the post even in the different epoch,
      * but the user is only allowed to vote either upvote or downvote, not both.
      *
-     * @param req express request, the body should contain postId, voteAction(refer to types/VoteAction.ts) and zkp
-     * @param res express response
+     * @param postId
+     * @param voteAction
+     * @param publicSignals
+     * @param proof
+     * @param onCancel
```

Comment:
Can you explain what `onCancel` is used for?

Reply:
removed it